src/ucs/datastruct/callbackq.h

```diff
@@ -186,6 +210,43 @@ ucs_status_t ucs_callbackq_remove_safe(ucs_callbackq_t *cbq, ucs_callback_t cb,
 
 
 /**
+ * Add a callback to the slow path list.
+ * This can be used from any context and any thread, including but not limited to:
+ * - A callback can add another callback.
+ * - A thread can add a callback while another thread is dispatching callbacks.
+ *
+ * Important note: It is not allowed to add the same element more than once.
```

Comment:
We will have to add this flag to release version, right? We wanted to avoid that to keep element size as small as possible. Or we can try to set list element fields to NULL during initialization.. But then we would need to update list API to set item fields to NULL when it is removed from the list.